pkg/autoupdate/autoupdate.go

```diff
@@ -33,8 +33,9 @@ const (
 )
 
 const (
-	DefaultMirror = "https://dl.kolide.co"
-	DefaultNotary = "https://notary.kolide.co"
+	DefaultMirror       = "https://dl.kolide.co"
+	DefaultNotary       = "https://notary.kolide.co"
+	DefaultNotaryPrefix = "kolide/"
```

Comment:
If you're using `path.Join`, skip the `/`
```suggestion
	DefaultNotaryPrefix = "kolide"
```